website/source/docs/providers/aws/r/elasticache_cluster.html.markdown

```diff
@@ -60,6 +60,10 @@ names to associate with this cache cluster
 * `security_group_ids` – (Optional, VPC only) One or more VPC security groups associated
  with the cache cluster
 
+* `apply_immediately` - (Optional) Specifies whether any database modifications
+     are applied immediately, or during the next maintenance window. Default is
+     `false`. See [Amazon ElastiCache Documentation for more information.][1]
```

Comment:
Can we add a (available since v0.6.0) in here. I'm not sure where.... maybe just at the end.